ParisBNF/abb/BNFabb95.xml

```diff
@@ -0,0 +1,483 @@
+                                  <dim unit="leaf">8</dim>
+                                  <locus from="104r" to="111v"/>
+                                  s.l.: 1, no stub
+                                  s.l.: 8, no stub
+                               </item>
+                              </list>
+                        </collation>
+                        <condition key="good"><ref target="#32"/> is mutilated (two columns are cut off),
+                      <ref target="#48"/> seems to have been composed out of two pieces of parchment sewn together.
+                    Some quires are reinforced with parchment guards.</condition>
+                     </supportDesc>
+
+                     <layoutDesc>
+                        <layout columns="3" writtenLines="25">
+                            <locus from="3r" to="72r"/>
+                        <ab type="pricking">Pricking and ruling are visible.</ab>
+                        <ab type="ruling" subtype="pattern">1A-1A-1A-1A-1A1A/0-0/0-0/C</ab>
+
+                      </layout>
+                     </layoutDesc>
+
+                  </objectDesc>
+                  <handDesc>
+                   <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script">18th century.</seg>
+                             <seg type="ink">Black, red.</seg>
+
+                             <desc>Irregular and hasty; by a mediocre scribe.
+                               The script is exceptionally small and nearly impossible to
+                               to read. The 5th-order markers are
+                             open, the length of the 3rd-order markers is emphasized.  </desc>
+                             </handNote>
+
+                             <handNote xml:id="h2" script="Ethiopic" corresp="#a1">
+                                  <seg type="script">First half of the 19th
+                                    century.</seg>
+                                       <seg type="ink">Black, red.</seg>
+                                       <date notBefore="1800" notAfter="1860"></date>
+                                       <desc>Somewhat irregular, careful; by a mediocre scribe.</desc>
+                                       </handNote>
+
+                  </handDesc>
+
+                  <decoDesc>
+
+                     <decoNote type="drawing" xml:id="d1">
+                        <locus target="#111r"/>
+                        Talismanic drawing of a cross and intersecting curved lines
+                        with loops at the end (stylized monograpms?).<desc>
+                      </desc>
+                     </decoNote>
+               </decoDesc>
+                  <additions>
+                    <list>
+                      <item xml:id="a1">
+                        <locus target="#1r"/>
+                        <desc type="Unclear">The note is Pater Noster in Coptic Bohairic transcribed in
```

Comment:
Maybe the ID could be used, what do you think? Especially as people might also search for "Our Father" and won't find it then